### ISSUE TITLE
"Cannot allocate memory" error when restarting OrthoFinder using -fg option

### ISSUE THREAD

**Issue by gauravdiwan89** [davidemms/OrthoFinder#562]
Hello!

I'm starting OrthoFinder for 508 proteomes with previously calculated BLAST results using the "-fg" option. I am running the program on a cluster where I can specify the amount of memory I can allocate. I used the suggestions in the user manual and allocated 64GB while using just 2 CPUs (64GB > 508 * 2 * 0.05GB). The 0.05GB is just a rough average since I have a mixture of proteomes from across the tree of life.

Unfortunately I run into a issue of memory allocation with the following Traceback:
```
Analysing Orthogroups
=====================

Calculating gene distances
--------------------------
Traceback (most recent call last):
  File "orthofinder.py", line 7, in <module>
  File "scripts_of/__main__.py", line 1788, in main
  File "scripts_of/__main__.py", line 1545, in GetOrthologues
  File "scripts_of/orthologues.py", line 999, in OrthologuesWorkflow
  File "scripts_of/orthologues.py", line 455, in RunAnalysis
  File "scripts_of/orthologues.py", line 299, in GetOGMatrices_FullParallel
  File "multiprocessing/__init__.py", line 260, in Array
  File "multiprocessing/sharedctypes.py", line 115, in Array
  File "multiprocessing/sharedctypes.py", line 83, in RawArray
  File "multiprocessing/sharedctypes.py", line 63, in _new_value
  File "multiprocessing/heap.py", line 243, in __init__
  File "multiprocessing/heap.py", line 223, in malloc
  File "multiprocessing/heap.py", line 120, in _malloc
  File "multiprocessing/heap.py", line 82, in __init__
mmap.error: [Errno 12] Cannot allocate memory
[1173] Failed to execute script orthofinder
```

Does this error arise because I have allocated too little memory? Also, what is the best way to accurately estimate the amount of memory I should allocate (apart from what is in the user manual)?

Thanks!

**Comment by davidemms** [comment]
Hi

Sorry, those figures were for the first stage of the algorithm, I'll update the manual to make that clearer. If you are having RAM issues for the tree inference you may find that MSA trees using -"M msa" may use less RAM. If you're able to allocate a lot more RAM though that would probably be best.

Best wishes
David

**Comment by gauravdiwan89** [comment]
Thanks for the clarification, David. I had to indeed allocate more memory for the program to run. It did successfully run and finish! 
I would just like to point out that using the multiple cpu option `-a 2` or more always gave warnings and errors (the same as #526) and in the end I had to finish the run using a single cpu. As clarified in #526 the error occurs when one uses OrthoFinder_glibc-2.15.tar.gz which is what I used as well.
